config/puma.rb

```diff
@@ -4,7 +4,7 @@
 
 preload_app!
 
-rackup      DefaultRackup
+rackup      DefaultRackup if defined?(DefaultRackup)
```

Comment:
Certainly, without this fix, we get the following error.
I don't understand what is happening...

```
config/puma.rb:7:in `_load_from': uninitialized constant Puma::DSL::DefaultRackup (NameError)
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/puma-6.2.2/lib/puma/dsl.rb:132:in `instance_eval'
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/puma-6.2.2/lib/puma/dsl.rb:132:in `_load_from'
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/puma-6.2.2/lib/puma/configuration.rb:237:in `block in load'
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/puma-6.2.2/lib/puma/configuration.rb:237:in `each'
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/puma-6.2.2/lib/puma/configuration.rb:237:in `load'
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/puma-6.2.2/lib/puma/launcher.rb:54:in `initialize'
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/puma-6.2.2/lib/rack/handler/puma.rb:68:in `new'
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/puma-6.2.2/lib/rack/handler/puma.rb:68:in `run'
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/sinatra-2.2.4/lib/sinatra/base.rb:1587:in `start_server'
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/sinatra-2.2.4/lib/sinatra/base.rb:1520:in `run!'
	from /home/daipom/work/fluentd/fluentd-website/vendor/ruby/3.2.0/gems/sinatra-2.2.4/lib/sinatra/main.rb:45:in `block in <module:Sinatra>'
```

Reply:
This configuration is ducumented by Heroku and it seems be like the committed code: https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#config

`DefaultRackup` was removed in puma 6.0: https://github.com/puma/puma/pull/2928

Reply:
I see! Thanks!